pkg/kubelet/util/sliceutils/sliceutils.go

```diff
@@ -53,6 +53,9 @@ func (s PodsByCreationTime) Less(i, j int) bool {
 type ByImageSize []kubecontainer.Image
 
 func (a ByImageSize) Less(i, j int) bool {
```

Comment:
Do we have unit testing for this function? If not, could we add it? If yes, could we update it to capture this new use case?

Reply:
Unit test added.